manifests/plugin/sysevent.pp

```diff
@@ -0,0 +1,64 @@
+#== Class: collectd::plugin::sysevent
+#
+# Class to manage sysevent plugin for collectd
+#
+# Documentation:
+#   https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_sysevent
+#
+# === Parameters
+#
+# [*ensure*]
+#  Ensure param for collectd::plugin type.
+#  Defaults to 'ensure'
+#
+# [*manage_package*]
+#  Set to true if Puppet should manage plugin package installation.
+#  Defaults to $collectd::manage_package
+#
+# [*listen_host*]
+#  Listen on this IP for incoming rsyslog messages.
+#  Defaults to '127.0.0.1'
+#
+# [*listen_port*]
+#  Listen on this port for incoming rsyslog messages.
+#  Defaults to 6666
+#
+# [*regex_filter*]
+#  Enumerate a regex filter to apply to all incoming rsyslog messages.  If a
+#  message matches this filter, it will be published.
+#  Defaults to '.*'
+#
+# [*buffer_size*]
+#  Maximum allowed size for incoming rsyslog messages.  Messages that exceed
+#  this number will be truncated to this size.  Default is 4096 bytes.
+#  Defaults to undef
+#
+# [*buffer_length*]
+#  Maximum number of rsyslog events that can be stored in plugin's ring buffer.
+#  Once an event has been read, its location becomes available for storing
+#  a new event.
+#  Defaults to undef
+#
+class collectd::plugin::sysevent (
+  Enum['present', 'absent'] $ensure  = 'present',
+  Boolean $manage_package            = $collectd::manage_package,
+  Stdlib::Host $listen_host          = '127.0.0.1',
+  Integer $listen_port               = 6666,
```

Comment:
Can you use this type please? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port.pp. We then need to make sure that stdlib 4.25.0 is the lowest supported version in our metadata.json

Reply:
Will do.